list_resources.sh

```diff
@@ -4,7 +4,7 @@ cd `dirname $0`
 for file in `ls ./specs`; do
 	work_dir=./specs/$file
 	if [ -d $work_dir ]; then
-		if [ -n "$(git status -s $work_dir)" ];
+		if [ -n "$(git diff --name-only remotes/origin/master remotes/origin/$source_branch -- $work_dir)" ];
```

Comment:
What if we need to run it in local environment ? Can we add a fallback logic ?

Reply:
Add the fallback logic in script.

- We can run `make build SOURCE_BRANCH=branchname`, to generate codes for new added/modified resources in local branch 
- If `SOURCE_BRANCH` is not set, it uses current branch as default, and generates codes for new added/modified resources in current branch
- For uncommitted changes, just uses `make build RESOURCE=resourcename` as before